test/contentset.test.js

```diff
@@ -0,0 +1,200 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const { codes } = require('../src/SDKErrors')
+
+beforeEach(() => {
+    fetchMock.reset()
+})
+test('createContentSet - program-not-found', async () => {
```

Comment:
I've mentioned this in a previous comment as well: we could leverage a proper test structure, even though the old code from this project doesn't follow those rules.

```
describe('Content Set', () => {
     beforeEach(() => { ... } };

     describe('Create', () => {
           test('should successfully create content set', async () => {
                  ....
            });
            test('should return program not found', async () => {
                  ....
            });

            test('should return Bad Request', async () => {
                  ....
            })
     });

      describe('Delete', () => {
            test('scenario', async () => {
                  ....
            });

            test('scenario', async () => {
                  ....
            })
     });
});
```

Similar for Content Flow.

And for file names, the pattern that I'm seeing used for the other flows is kebab-case:
`contentflow.test.js` -> `content-flow.test.js`
`contentset.test.js` -> `content-set.test.js`